<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMedicineEdit</class>
 <widget class="QDialog" name="CMedicineEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование лекарств</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbMedName">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMedName"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbDateBay">
       <property name="text">
        <string>Дата покупки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditMedDateBay"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbMedPlace">
       <property name="text">
        <string>Место хранения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMedPlace"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lbEditMedLife">
       <property name="text">
        <string>Срок годности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditMedLife"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxInstruction">
     <property name="title">
      <string>Инструкция</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEditMedInstruction"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMedSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMedCansel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonMedCansel</sender>
   <signal>clicked()</signal>
   <receiver>CMedicineEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
